infor-design/enterprise-wc
# 1329 - IdsDataGrid vscroll selection

**Explain the details for making this change. What existing problem does the pull request solve?**
Fixes storing selected/deselected state when using virtual scroll in data grid

**Related github/jira issue (required)**:
Closes #1329 

**Steps necessary to review your pull request (required)**:
1. Checkout branch, `npm install`, `npm run start`
2. Go to http://localhost:4300/ids-data-grid/virtual-scroll-row-start.html
3. Select/Deselect some rows
4. Scroll down/up until those rows aren't in DOM
5. Scroll back to those selected/deslected rows and make sure they kept their state
6. Check that other data grid scroll features/examples are unaffected by changes

**Testing Programmatically**
1. Call `$('ids-data-grid').selectRow( [index] )` for rows not in DOM
2. Scroll to row and check that it's selected
3. Test similarly with `$('ids-data-grid').deSelectRow( [index] )`

**Included in this Pull Request**:
